src/modules/data/modelData_gltf.c

```diff
@@ -888,7 +890,7 @@ ModelData* lovrModelDataInitGltf(ModelData* model, Blob* source, ModelDataIO* io
           scale[2] = NOM_FLOAT(json, token);
         } else if (STR_EQ(key, "name")) {
           gltfString name = NOM_STR(json, token);
-          map_set(&model->nodeMap, hash64(name.data, name.length), model->nodeCount - i);
+          map_set(&model->nodeMap, hash64(name.data, name.length), model->nodeCount - extraNodes - i);
```

Comment:
I wonder if `node - model->nodes` would be a more clear/robust way to write this without needing `extraNodes`.

Reply:
Should we apply the same to `model->textures` and `model->materials` then?

Reply:
Added